app/_locales/en/messages.json

```diff
@@ -2792,6 +2792,15 @@
   "notifications23Title": {
     "message": "Stay safe with security alerts"
   },
+  "notifications24ActionText": {
+    "message": "Got it"
+  },
+  "notifications24Description": {
+    "message": "Advanced gas fee settings are now remembered based on the network you're using. This means you can set specific advanced gas fees for each network and avoid overpaying for gas or stuck transactions."
```

Comment:
Should we be replacing all 8 instances of "you're" in our messages? Is there a reason for this other than preference?

Reply:
cc @coreyjanssen what are your thoughts?

Reply:
Our standard is that our voice should be simple and concise, and a what's new message is a pretty low stress time in the scope of tone (unlike an error message or warning). We often go for conjugations like you're instead of you are in instances like this one, but if __you_ are_ vs. _you're_ is causing friction for the user that I'm not seeing here, I'm cool with opting for _you are_ in this case! @brad-decker @legobeat

Reply:
Lets keep it as is, and then if we decide in the future to revise our handling of conjugations we can fix all cases at that point.